java/mapreduce/src/test/org/apache/orc/mapreduce/TestMapreduceOrcOutputFormat.java

```diff
@@ -153,6 +155,34 @@ public void testColumnSelection() throws Exception {
     assertEquals(false, reader.nextKeyValue());
   }
 
+  @Test
+  public void testAcidSelectionNoSchema() throws IOException, InterruptedException {
+    TaskAttemptID id = new TaskAttemptID("jt", 0, TaskType.MAP, 0, 1);
+    TaskAttemptContext attemptContext = new TaskAttemptContextImpl(conf, id);
+    // struct<operation:int,originalTransaction:bigint,bucket:int,rowId:bigint,currentTransaction:bigint,
+    // row:struct<i:int,j:int,k:int>>
```

Comment:
I verified that the newly added test file has 5000 rows with this schema correctly.